paywall/src/components/content/CheckoutContent.tsx

```diff
@@ -38,8 +40,6 @@ export const Wrapper: React.FunctionComponent<WrapperProps> = ({
     })
   }, [postMessage])
 
-  const paywallConfig = usePaywallConfig()
```

Comment:
Can you add a comment inside of `usePaywallConfig` saying that calling it twice WILL create issues?

Reply:
so that we don't end up in the same situation soon...